app/models/manageiq/providers/ansible_tower/shared/automation_manager/job.rb

```diff
@@ -31,6 +32,23 @@ def db_name
     def status_class
       "#{self.name}::Status".constantize
     end
+
+    private
+
+    def filter_options(template, options)
```

Comment:
@bzwei 
Can we name this function to indicate its purpose.
e.g. transform_extra_vars_keys or convert_extra_vars_keys